src/test/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerIssueDecisionTest.java

```diff
@@ -49,42 +44,23 @@ void shouldAddConfigurationToConfigBuilder() {
             .contains(CASEWORKER_ISSUE_DECISION);
     }
 
-    @Test
-    void shouldSuccessfullyAddDecisionRecipients() {
-        //Given
-        final CaseData caseData = caseData();
-        final CaseIssueDecision decision = caseData.getCaseIssueDecision();
-        Set<SubjectCIC> subjectSet = new HashSet<>();
-        subjectSet.add(SubjectCIC.SUBJECT);
-        decision.setRecipientSubject(subjectSet);
-
-        Set<RepresentativeCIC> representativeSet = new HashSet<>();
-        representativeSet.add(RepresentativeCIC.REPRESENTATIVE);
-        decision.setRecipientRepresentative(representativeSet);
-
-        Set<RespondentCIC> respondentSet = new HashSet<>();
-        respondentSet.add(RespondentCIC.RESPONDENT);
-        decision.setRecipientRespondent(respondentSet);
-
-        //Then
-        assertThat(caseData.getCaseIssueDecision().getRecipientSubject().contains(SUBJECT));
-        assertThat(caseData.getCaseIssueDecision().getRecipientRepresentative().contains(REPRESENTATIVE));
-        assertThat(caseData.getCaseIssueDecision().getRecipientRespondent().contains(RESPONDENT));
-    }
-
     @Test
     void shouldSetState() {
         //Given
         final CaseDetails<CaseData, State> details = new CaseDetails<>();
         final CaseDetails<CaseData, State> beforeDetails = new CaseDetails<>();
         final CaseData caseData = caseData();
+        final CaseIssueDecision decision = caseData.getCaseIssueDecision();
+        Set<ContactPartiesCIC> subjectSet = new HashSet<>();
+        subjectSet.add(ContactPartiesCIC.SUBJECTTOCONTACT);
+        decision.setRecipients(subjectSet);
         details.setData(caseData);
 
         //When
         AboutToStartOrSubmitResponse<CaseData, State> response = issueDecision.aboutToSubmit(details, beforeDetails);
 
         //Then
-        assertThat(response.getState())
-            .isEqualTo(CaseManagement);
+        assertThat(response.getState()).isEqualTo(CaseManagement);
+        assertThat(response.getData().getCaseIssueDecision().getRecipients().equals(ContactPartiesCIC.SUBJECTTOCONTACT));
     }
```

Comment:
This is an enum, so use == instead of .equals, as it's checked at compile time and == is null safe.
Viz.
`assertThat(response.getData().getCaseIssueDecision().getRecipients() == ContactPartiesCIC.SUBJECTTOCONTACT);`

Reply:
OK. If it returns a Set<ContactPartiesCIC>, instead of equals, use contains?